providers/standard/src/airflow/providers/standard/operators/trigger_dagrun.py

```diff
@@ -223,6 +223,8 @@ def execute(self, context: Context):
 
         if self.fail_when_dag_is_paused:
             dag_model = DagModel.get_current(self.trigger_dag_id)
+            if not dag_model:
+                raise AirflowException(f"Dag {self.trigger_dag_id} is not found")
```

Comment:
```suggestion
                raise ValueError(f"Dag {self.trigger_dag_id} is not found")
```

Based on previous discussion in the dev list, we should avoid raising AirflowExecption.

Reply:
I am raising the same kind of exception we are already raising one line below. If we decide to change that, we should do it altogether

Reply:
We are trying not to raise new occurrences anymore, it is raised below yes, but that was added a long time ago, I have a suggestion below which could avoid raising the exception